### PR TITLE
Abb Powerone document new energy sensor

### PR DESCRIPTION
**Pull request by davet2001**
## Proposed change

Add the extra energy metering sensor to the documentation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/58454
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/aurora_abb_powerone.markdown [hunk 1]
```diff
@@ -20,14 +20,19 @@ The inverter was formerly made by PowerOne who got taken over by ABB.
...
+Note the PV inverter will be unresponsive to communications when in darkness, 
+so the sensors will report 'Unavailable' during the night.
+
+The RS485 connection can be made using a low-cost USB-RS485 converter. It works using a 2-wire interface but an interface with a separate ground reference may be more reliable.
 
 ## Configuration
 
+Configuration should be done via the user interface. The inverter will need to be on (i.e. in daylight) and connected correctly in order to do the first-time setup. Normally it is sufficient to select the correct serial port and leave the default address `2`.
+
+Configuration via `configuration.yaml` is now deprecated but does still work.
```
Comment (frenck) [inline source/_integrations/aurora_abb_powerone.markdown]: We don't document deprecations, we remove them from the documentation.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/aurora_abb_powerone.markdown (hunk 1 above)
Comment (davet2001) [reply source/_integrations/aurora_abb_powerone.markdown]: Done

**Comment by davet2001** [comment]
Comments addressed, but #13748 should be merged first.

**Comment by davet2001** [comment]
I think that since the parent has been [cherry picked](https://github.com/home-assistant/core/pull/58454#event-5560649970) into 2021.11.x, this needs to be merged (need to remove the `next` label).
(The [live documentation](https://www.home-assistant.io/integrations/aurora_abb_powerone/) only refers to 2 sensors but there are now 3).

**Review by frenck — APPROVED** [review]
Thanks, @davet2001 👍

**Pull request merged**